examples/output/2island/geomorphAreaOverlap.json

```diff
@@ -73,8 +73,8 @@
       "sketchId": "17853",
       "classId": "escarpments",
       "groupId": null,
-      "geographyId": null,
-      "value": 110960538.94313982,
+      "geographyId": "nearshore",
+      "value": 110960012.640865,
```

Comment:
Some of these geomorph values reduced slightly.  I think this came from the recent addition of the double count fix to overlapFeatures in the gp library